src/pint/models/model_builder.py

```diff
@@ -413,6 +430,16 @@ def choose_model(self, param_inpar):
             Dictionary of the unique parameters in .par file with the key is the
             parfile line. :func:`parse_parfile` returns this dictionary.
 
+        allow_T2 : True, False, or str optional
+            Whether to convert a T2 binary model to an appropriate underlying
+            binary model. Default is False, and will throw an error upon
+            encountering the T2 binary model. If True, the binary model will be
+            converted to the most appropriate PINT-compatible binary model.
```

Comment:
What does an str value for this parameter do?

Reply:
Changing the docstring